tests/p2p/dumb_peer.py

```diff
@@ -0,0 +1,23 @@
+from p2p import eth
+from p2p.peer import (
+    BasePeer,
+)
+
+from p2p import protocol
+
+
+class DumbPeer(BasePeer):
+    _supported_sub_protocols = [eth.ETHProtocol]
+    sub_proto = None  # type: eth.ETHProtocol
+    head_td = None  # type: int
+    head_hash = None  # type: bytes
```

Comment:
Why not subclass `ETHPeer`?

Reply:
I thought both works, but `ETHPeer` does make sense.   👍 Will update it.